src/test/scala/org/constellation/checkpoint/CheckpointAcceptanceServiceTest.scala

```diff
@@ -0,0 +1,23 @@
+package org.constellation.checkpoint
+
+import org.constellation.{DAO, TestHelpers}
+import org.constellation.p2p.PeerData
+import org.constellation.schema.Id
+import org.mockito.{ArgumentMatchersSugar, IdiomaticMockito}
+import org.mockito.cats.IdiomaticMockitoCats
+import org.scalatest.{BeforeAndAfter, FunSuite, Matchers}
+
+class CheckpointAcceptanceServiceTest
+    extends FunSuite
+    with ArgumentMatchersSugar
+    with BeforeAndAfter
+    with IdiomaticMockito
+    with IdiomaticMockitoCats
+    with Matchers {
+
+  private val readyFacilitators: Map[Id, PeerData] = TestHelpers.prepareFacilitators(1)
+  private val dao: DAO = TestHelpers.prepareRealDao(readyFacilitators)
+
+  test("should accept transactions") {}
```

Comment:
Adding comment here about `CheckpointAcceptanceService`: we will want to modify `areTransactionsAllowedForAcceptance` and `isCheckpointBlockAllowedForAcceptance` to call `doubleSpendValidator` and `blacklistAddressValidator`, so that a node accepting a checkpoint can go through the double spend flow.